src/client/nav/response.js

```diff
@@ -582,8 +582,11 @@ spf.nav.response.extract_ = function(frag) {
           var url = attr.match(spf.nav.response.AttributeRegEx.SRC);
           url = url ? url[1] : '';
           var async = spf.nav.response.AttributeRegEx.ASYNC.test(attr);
-          result['scripts'].push(
-              {url: url, text: text, name: name, async: async});
+          var json = spf.nav.response.AttributeRegEx.JSON.test(attr);
```

Comment:
I think this is important, but I have a couple of issues.

1) As is, this'll drop the script entirely. It's probably a fine first step, as it's pretty close to the current behavior, but we probably need to allow these to be available in some capacity, either in the DOM or in some other structure. Since you presumably have a use case, thoughts?
2) This is probably better off as a Whitelist of valid types instead of a blacklist. How about only allowing scripts with empty types or type='text/javascript'?
3) Please add tests.